Networking/NetworkingTests/Remote/StripeRemoteTests.swift

```diff
@@ -0,0 +1,309 @@
+import XCTest
+@testable import Networking
+
+
+/// Stripe Remote Unit Tests
+///
+final class StripeRemoteTests: XCTestCase {
+
+    /// Dummy Network Wrapper
+    ///
+    let network = MockNetwork()
+
+    /// Dummy Site ID
+    ///
+    let sampleSiteID: Int64 = 1234
```

Comment:
nit: could these be private?
```suggestion
    private let network = MockNetwork()

    /// Dummy Site ID
    ///
    private let sampleSiteID: Int64 = 1234
```

Reply:
Fixed in https://github.com/woocommerce/woocommerce-ios/pull/5799/commits/e7947eefd32083ab3117916279d31c03bf6a5e93 in both this test and the original test this was based on